<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<html lang="en">
  <head>
    <meta charset="utf-8">
<!--
    <link rel="stylesheet" type="text/css" href="http://epigenomegateway.wustl.edu/browser/style.css" />
    <script type="text/javascript" src="http://epigenomegateway.wustl.edu/browser/js/base.js"></script>
    <script type="text/javascript" src="http://epigenomegateway.wustl.edu/browser/js/personality.js"></script>
    <script type="text/javascript" src="http://epigenomegateway.wustl.edu/browser/js/embed.js"></script>
-->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    
    <link href="./report/jquery.treetable.css" rel="stylesheet" type="text/css" />
    <link href="./report/jquery.treetable.theme.default.css" rel="stylesheet" />
    <script src="./report/jquery.treetable.js"></script>

    <script type="text/javascript">
      $( function(){
      	$("#filetable").treetable({ 
      		expandable: true,
      		onNodeCollapse: function() {
      		},
      		onNodeExpand: function() {
      		}
      	});
      });
    </script>
  </head>
  <body>
<div id='latest_git_commit'><b>Pipeline version</b> (latest git commit SHA1):<a target=_blank, href=https://github.com/kundajelab/atac_dnase_pipelines/commit/9602e73764c8dde05ef43a1161009bb012f49fc6>9602e73764c8dde05ef43a1161009bb012f49fc6</a> (Wed Mar 22 11:34:04 2017)</div><br>
<div id='dirs_and_files'><b>Directories and files</b><table id='filetable'> <caption><a href='#' onclick="jQuery('#filetable').treetable('expandAll'); return false;">Expand all</a> &nbsp&nbsp<a href='#' onclick="jQuery('#filetable').treetable('collapseAll'); return false;">Collapse all</a></caption> <thead><tr><th>Files</th><th>Path</th></tr></thead> <tbody><tr data-tt-id='Alignment'><td> Alignment </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Alignment/Replicate 1' data-tt-parent-id='Alignment'><td> Replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Filtered & deduped bam' data-tt-parent-id='Alignment/Replicate 1'><td> Filtered & deduped bam </td><td><a href='./align/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.bam' target='_blank'>./align/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.bam</a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Fastq 1' data-tt-parent-id='Alignment/Replicate 1'><td> Fastq 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Fastq 2' data-tt-parent-id='Alignment/Replicate 1'><td> Fastq 2 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Trimmed fastq 1' data-tt-parent-id='Alignment/Replicate 1'><td> Trimmed fastq 1 </td><td><a href='./align/rep1/scATAC.Combined_R1_Sequences.trim.fastq.gz' target='_blank'>./align/rep1/scATAC.Combined_R1_Sequences.trim.fastq.gz</a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Trimmed fastq 2' data-tt-parent-id='Alignment/Replicate 1'><td> Trimmed fastq 2 </td><td><a href='./align/rep1/scATAC.Combined_R2_Sequences.trim.fastq.gz' target='_blank'>./align/rep1/scATAC.Combined_R2_Sequences.trim.fastq.gz</a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Bam' data-tt-parent-id='Alignment/Replicate 1'><td> Bam </td><td><a href='./align/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.bam' target='_blank'>./align/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.bam</a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Tag-align' data-tt-parent-id='Alignment/Replicate 1'><td> Tag-align </td><td><a href='./align/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.tagAlign.gz' target='_blank'>./align/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.tagAlign.gz</a><br></td></tr><tr data-tt-id='Alignment/Pseudo-replicates' data-tt-parent-id='Alignment'><td> Pseudo-replicates </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Alignment/Pseudo-replicates/Replicate 1' data-tt-parent-id='Alignment/Pseudo-replicates'><td> Replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Alignment/Pseudo-replicates/Replicate 1/Pseudo-replicate 1' data-tt-parent-id='Alignment/Pseudo-replicates/Replicate 1'><td> Pseudo-replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Alignment/Pseudo-replicates/Replicate 1/Pseudo-replicate 1/Tag-align' data-tt-parent-id='Alignment/Pseudo-replicates/Replicate 1/Pseudo-replicate 1'><td> Tag-align </td><td><a href='./align/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.tagAlign.gz' target='_blank'>./align/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.tagAlign.gz</a><br></td></tr><tr data-tt-id='Alignment/Pseudo-replicates/Replicate 1/Pseudo-replicate 2' data-tt-parent-id='Alignment/Pseudo-replicates/Replicate 1'><td> Pseudo-replicate 2 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Alignment/Pseudo-replicates/Replicate 1/Pseudo-replicate 2/Tag-align' data-tt-parent-id='Alignment/Pseudo-replicates/Replicate 1/Pseudo-replicate 2'><td> Tag-align </td><td><a href='./align/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.tagAlign.gz' target='_blank'>./align/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.tagAlign.gz</a><br></td></tr><tr data-tt-id='Signal tracks'><td> Signal tracks </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Signal tracks/MACS2' data-tt-parent-id='Signal tracks'><td> MACS2 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 1' data-tt-parent-id='Signal tracks/MACS2'><td> Replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 1/P-value' data-tt-parent-id='Signal tracks/MACS2/Replicate 1'><td> P-value </td><td><a href='./signal/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval.signal.bigwig' target='_blank'>./signal/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval.signal.bigwig</a><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 1/Fold enrichment' data-tt-parent-id='Signal tracks/MACS2/Replicate 1'><td> Fold enrichment </td><td><a href='./signal/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.fc.signal.bigwig' target='_blank'>./signal/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.fc.signal.bigwig</a><br></td></tr><tr data-tt-id='Peaks'><td> Peaks </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/MACS2' data-tt-parent-id='Peaks'><td> MACS2 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1' data-tt-parent-id='Peaks/MACS2'><td> Replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1/Gapped peak (p-val thresh=.01)' data-tt-parent-id='Peaks/MACS2/Replicate 1'><td> Gapped peak (p-val thresh=.01) </td><td><a href='./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.gappedPeak.gz' target='_blank'>./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.gappedPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1/Narrow peak (p-val thresh=.01)' data-tt-parent-id='Peaks/MACS2/Replicate 1'><td> Narrow peak (p-val thresh=.01) </td><td><a href='./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.narrowPeak.gz' target='_blank'>./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1/Narrow peak' data-tt-parent-id='Peaks/MACS2/Replicate 1'><td> Narrow peak </td><td><a href='./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.narrowPeak.gz' target='_blank'>./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1/Gapped peak' data-tt-parent-id='Peaks/MACS2/Replicate 1'><td> Gapped peak </td><td><a href='./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.gappedPeak.gz' target='_blank'>./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.gappedPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1/Filtered narrow peak' data-tt-parent-id='Peaks/MACS2/Replicate 1'><td> Filtered narrow peak </td><td><a href='./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.500K.narrowPeak.gz' target='_blank'>./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.500K.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1/Filtered gapped peak' data-tt-parent-id='Peaks/MACS2/Replicate 1'><td> Filtered gapped peak </td><td><a href='./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.500K.gappedPeak.gz' target='_blank'>./peak/macs2/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.500K.gappedPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates' data-tt-parent-id='Peaks/MACS2'><td> Pseudo-replicates </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates'><td> Replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1'><td> Pseudo-replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1/Narrow peak' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1'><td> Narrow peak </td><td><a href='./peak/macs2/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.pf.pval0.1.narrowPeak.gz' target='_blank'>./peak/macs2/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.pf.pval0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1/Gapped peak' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1'><td> Gapped peak </td><td><a href='./peak/macs2/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.pf.pval0.1.gappedPeak.gz' target='_blank'>./peak/macs2/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.pf.pval0.1.gappedPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1/Filtered narrow peak' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1'><td> Filtered narrow peak </td><td><a href='./peak/macs2/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.pf.pval0.1.500K.narrowPeak.gz' target='_blank'>./peak/macs2/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.pf.pval0.1.500K.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1/Filtered gapped peak' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 1'><td> Filtered gapped peak </td><td><a href='./peak/macs2/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.pf.pval0.1.500K.gappedPeak.gz' target='_blank'>./peak/macs2/pseudo_reps/rep1/pr1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr1.tn5.pf.pval0.1.500K.gappedPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1'><td> Pseudo-replicate 2 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2/Narrow peak' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2'><td> Narrow peak </td><td><a href='./peak/macs2/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.pf.pval0.1.narrowPeak.gz' target='_blank'>./peak/macs2/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.pf.pval0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2/Gapped peak' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2'><td> Gapped peak </td><td><a href='./peak/macs2/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.pf.pval0.1.gappedPeak.gz' target='_blank'>./peak/macs2/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.pf.pval0.1.gappedPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2/Filtered narrow peak' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2'><td> Filtered narrow peak </td><td><a href='./peak/macs2/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.pf.pval0.1.500K.narrowPeak.gz' target='_blank'>./peak/macs2/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.pf.pval0.1.500K.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2/Filtered gapped peak' data-tt-parent-id='Peaks/MACS2/Pseudo-replicates/Replicate 1/Pseudo-replicate 2'><td> Filtered gapped peak </td><td><a href='./peak/macs2/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.pf.pval0.1.500K.gappedPeak.gz' target='_blank'>./peak/macs2/pseudo_reps/rep1/pr2/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.pr2.tn5.pf.pval0.1.500K.gappedPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Naive overlap' data-tt-parent-id='Peaks/MACS2'><td> Naive overlap </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/MACS2/Naive overlap/Narrow peak' data-tt-parent-id='Peaks/MACS2/Naive overlap'><td> Narrow peak </td><td><a href='./peak/macs2/overlap/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.500K.naive_overlap.narrowPeak.gz' target='_blank'>./peak/macs2/overlap/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.500K.naive_overlap.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Naive overlap/Gapped peak' data-tt-parent-id='Peaks/MACS2/Naive overlap'><td> Gapped peak </td><td><a href='./peak/macs2/overlap/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.500K.naive_overlap.gappedPeak.gz' target='_blank'>./peak/macs2/overlap/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.tn5.pf.pval0.1.500K.naive_overlap.gappedPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/IDR' data-tt-parent-id='Peaks'><td> IDR </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/IDR/Pseudo-replicates' data-tt-parent-id='Peaks/IDR'><td> Pseudo-replicates </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/IDR/Pseudo-replicates/Replicate 1' data-tt-parent-id='Peaks/IDR/Pseudo-replicates'><td> Replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/IDR/Pseudo-replicates/Replicate 1/IDR peak' data-tt-parent-id='Peaks/IDR/Pseudo-replicates/Replicate 1'><td> IDR peak </td><td><a href='./peak/idr/pseudo_reps/rep1/COMBINED_TEST_rep1-pr.IDR0.1.narrowPeak.gz' target='_blank'>./peak/idr/pseudo_reps/rep1/COMBINED_TEST_rep1-pr.IDR0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/IDR/Optimal set' data-tt-parent-id='Peaks/IDR'><td> Optimal set </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/IDR/Optimal set/IDR peak' data-tt-parent-id='Peaks/IDR/Optimal set'><td> IDR peak </td><td><a href='./peak/idr/optimal_set/COMBINED_TEST_rep1-pr.IDR0.1.narrowPeak.gz' target='_blank'>./peak/idr/optimal_set/COMBINED_TEST_rep1-pr.IDR0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/IDR/Conservative set' data-tt-parent-id='Peaks/IDR'><td> Conservative set </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='Peaks/IDR/Conservative set/IDR peak' data-tt-parent-id='Peaks/IDR/Conservative set'><td> IDR peak </td><td><a href='./peak/idr/conservative_set/COMBINED_TEST_rep1-pr.IDR0.1.narrowPeak.gz' target='_blank'>./peak/idr/conservative_set/COMBINED_TEST_rep1-pr.IDR0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='QC and logs'><td> QC and logs </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1' data-tt-parent-id='QC and logs'><td> Replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Bowtie2 map. log' data-tt-parent-id='QC and logs/Replicate 1'><td> Bowtie2 map. log </td><td><a href='./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.align.log' target='_blank'>./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.align.log</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Dedup. log' data-tt-parent-id='QC and logs/Replicate 1'><td> Dedup. log </td><td><a href='./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.dup.qc' target='_blank'>./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.dup.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Bowtie2 map. flagstat log' data-tt-parent-id='QC and logs/Replicate 1'><td> Bowtie2 map. flagstat log </td><td><a href='./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.flagstat.qc' target='_blank'>./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.flagstat.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Cross-corr. plot' data-tt-parent-id='QC and logs/Replicate 1'><td> Cross-corr. plot </td><td><a href='./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.25M.cc.plot.pdf' target='_blank'>./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.25M.cc.plot.pdf</a><br></td></tr><tr data-tt-id='QC and logs/IDR' data-tt-parent-id='QC and logs'><td> IDR </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='QC and logs/IDR/IDR QC log' data-tt-parent-id='QC and logs/IDR'><td> IDR QC log </td><td><a href='./qc/COMBINED_TEST_IDR_final.qc' target='_blank'>./qc/COMBINED_TEST_IDR_final.qc</a><br></td></tr><tr data-tt-id='QC and logs/IDR/Pseudo-replicates' data-tt-parent-id='QC and logs/IDR'><td> Pseudo-replicates </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='QC and logs/IDR/Pseudo-replicates/Replicate 1' data-tt-parent-id='QC and logs/IDR/Pseudo-replicates'><td> Replicate 1 </td><td><a href='' target='_blank'></a><br></td></tr><tr data-tt-id='QC and logs/IDR/Pseudo-replicates/Replicate 1/FRiP' data-tt-parent-id='QC and logs/IDR/Pseudo-replicates/Replicate 1'><td> FRiP </td><td><a href='./peak/idr/pseudo_reps/rep1/COMBINED_TEST_rep1-pr.IDR0.1.narrowPeak.FRiP.qc' target='_blank'>./peak/idr/pseudo_reps/rep1/COMBINED_TEST_rep1-pr.IDR0.1.narrowPeak.FRiP.qc</a><br></td></tr><tr data-tt-id='QC and logs/IDR/Pseudo-replicates/Replicate 1/IDR plot' data-tt-parent-id='QC and logs/IDR/Pseudo-replicates/Replicate 1'><td> IDR plot </td><td><a href='./peak/idr/pseudo_reps/rep1/COMBINED_TEST_rep1-pr.unthresholded-peaks.txt.png' target='_blank'>./peak/idr/pseudo_reps/rep1/COMBINED_TEST_rep1-pr.unthresholded-peaks.txt.png</a><br></td></tr></tbody></table></div><br>
<div id='viz'><b>Visualization (add <i>-url_base [URL_ROOT_DIR_FOR_OUT_DIR]</i> to the command line.) </b><div style='padding-left: 20px'><a href=http://epigenomegateway.wustl.edu/browser/?genome=hg38_ENCODE&tknamewidth=275&datahub=/./COMBINED_TEST_tracks.json target='_blank'>Visualize</a>&nbsp&nbsp<a href=./COMBINED_TEST_tracks.json target='_blank'>JSON (datahub)</a></div></div><br>

<div id='graph'><b>Workflow diagram</b>&nbsp<small>(g. peak : gapped peak from MACS2, n. peak : narrow peak from MACS2, r. peak : region peak from SPP)</small><br><object data='./report/COMBINED_TEST_workflow.svg'></object></div><br></div><br>
<div id='atac_qc'><div id='table_Flagstat-(raw)-QC'><b>Flagstat (raw) QC</b><div style='padding-left: 20px'>
<table border='1'>
<tr><th nowrap align=left>Total</th><th nowrap align=left>Total(QC-failed)</th><th nowrap align=left>Dupes</th><th nowrap align=left>Dupes(QC-failed)</th><th nowrap align=left>Mapped</th><th nowrap align=left>Mapped(QC-failed)</th><th nowrap align=left>% Mapped</th><th nowrap align=left>Paired</th><th nowrap align=left>Paired(QC-failed)</th><th nowrap align=left>Read1</th><th nowrap align=left>Read1(QC-failed)</th><th nowrap align=left>Read2</th><th nowrap align=left>Read2(QC-failed)</th><th nowrap align=left>Properly Paired</th><th nowrap align=left>Properly Paired(QC-failed)</th><th nowrap align=left>% Properly Paired</th><th nowrap align=left>With itself</th><th nowrap align=left>With itself(QC-failed)</th><th nowrap align=left>Singletons</th><th nowrap align=left>Singletons(QC-failed)</th><th nowrap align=left>% Singleton</th><th nowrap align=left>Diff. Chroms</th><th nowrap align=left>Diff. Chroms (QC-failed)</th></tr>
<tr><td>2794222</td><td>0</td><td>0</td><td>0</td><td>1296704</td><td>0</td><td>46.41</td><td>2794222</td><td>0</td><td>1397111</td><td>0</td><td>1397111</td><td>0</td><td>803878</td><td>0</td><td>28.77</td><td>1015462</td><td>0</td><td>281242</td><td>0</td><td>10.07</td><td>66077</td><td>0</td></tr>
</table></div></div><br><div id='table_Dup.-QC'><b>Dup. QC</b><div style='padding-left: 20px'>
<table border='1'>
<tr><th nowrap align=left>Unpaired Reads</th><th nowrap align=left>Paired Reads</th><th nowrap align=left>Unmapped Reads</th><th nowrap align=left>Unpaired Dupes</th><th nowrap align=left>Paired Dupes</th><th nowrap align=left>Paired Opt. Dupes</th><th nowrap align=left>% Dupes</th></tr>
<tr><td>0</td><td>208311</td><td>0</td><td>0</td><td>62293</td><td>45621</td><td>0.299038</td></tr>
</table></div></div><br><div id='table_Flagstat-(filtered)-QC'><b>Flagstat (filtered) QC</b><div style='padding-left: 20px'>
<table border='1'>
<tr><th nowrap align=left>Total</th><th nowrap align=left>Total(QC-failed)</th><th nowrap align=left>Dupes</th><th nowrap align=left>Dupes(QC-failed)</th><th nowrap align=left>Mapped</th><th nowrap align=left>Mapped(QC-failed)</th><th nowrap align=left>% Mapped</th><th nowrap align=left>Paired</th><th nowrap align=left>Paired(QC-failed)</th><th nowrap align=left>Read1</th><th nowrap align=left>Read1(QC-failed)</th><th nowrap align=left>Read2</th><th nowrap align=left>Read2(QC-failed)</th><th nowrap align=left>Properly Paired</th><th nowrap align=left>Properly Paired(QC-failed)</th><th nowrap align=left>% Properly Paired</th><th nowrap align=left>With itself</th><th nowrap align=left>With itself(QC-failed)</th><th nowrap align=left>Singletons</th><th nowrap align=left>Singletons(QC-failed)</th><th nowrap align=left>% Singleton</th><th nowrap align=left>Diff. Chroms</th><th nowrap align=left>Diff. Chroms (QC-failed)</th></tr>
<tr><td>292036</td><td>0</td><td>0</td><td>0</td><td>292036</td><td>0</td><td>100.00</td><td>292036</td><td>0</td><td>146018</td><td>0</td><td>146018</td><td>0</td><td>292036</td><td>0</td><td>100.00</td><td>292036</td><td>0</td><td>0</td><td>0</td><td>0.00</td><td>0</td><td>0</td></tr>
</table></div></div><br><div id='table_PBC-QC'><b>PBC QC</b><div style='padding-left: 20px'>
<table border='1'>
<tr><th nowrap align=left>Total Read Pairs</th><th nowrap align=left>Distinct Read Pairs</th><th nowrap align=left>One Read Pair</th><th nowrap align=left>Two Read Pairs</th><th nowrap align=left>NRF = Distinct/Total</th><th nowrap align=left>PBC1 = OnePair/Distinct</th><th nowrap align=left>PBC2 = OnePair/TwoPair</th></tr>
<tr><td>153026</td><td>111288</td><td>82070</td><td>20718</td><td>0.727249</td><td>0.737456</td><td>3.961290</td></tr>
</table></div></div><br><div id='help-pbc'><p>NRF (non redundant fraction) <br>PBC1 (PCR Bottleneck coefficient 1) <br>PBC2 (PCR Bottleneck coefficient 2) <br>PBC1 is the primary measure. Provisionally <br><ul><li>0-0.5 is severe bottlenecking</li><li>0.5-0.8 is moderate bottlenecking </li><li>0.8-0.9 is mild bottlenecking </li><li>0.9-1.0 is no bottlenecking </li></ul></p></div><div id='table_Cross-corr.-QC'><b>Cross-corr. QC</b><div style='padding-left: 20px'>
<table border='1'>
<tr><th nowrap align=left>Reads</th><th nowrap align=left>Est. Fragment Len.</th><th nowrap align=left>Corr. Est. Fragment Len.</th><th nowrap align=left>Phantom Peak</th><th nowrap align=left>Corr. Phantom Peak</th><th nowrap align=left>Argmin. Corr.</th><th nowrap align=left>Min. Corr.</th><th nowrap align=left>NSC</th><th nowrap align=left>RSC</th></tr>
<tr><td>109129</td><td>0</td><td>0.00335468530055079</td><td>125</td><td>0.004161496</td><td>1500</td><td>0.002200196</td><td>1.524721</td><td>0.5886347</td></tr>
</table></div></div><br><div id='help-xcor'><p>Normalized strand cross-correlation coefficient (NSC) = col9 in outFile <br>Relative strand cross-correlation coefficient (RSC) = col10 in outFile <br>Estimated fragment length = col3 in outFile, take the top value <br>Important columns highlighted, but all/whole file can be stored for display <br></p><br><figure style='display: inline-block;'><img src='./qc/rep1/scATAC.Combined_R1_Sequences.trim.PE2SE.nodup.25M.cc.plot.png' width='500'><figcaption style='text-align: center;'><b>rep1</b></figcaption></figure>&nbsp<div id='table_IDR-QC'><b>IDR QC</b><div style='padding-left: 20px'>
<table border='1'>
<tr><th nowrap align=left>Nt</th><th nowrap align=left>N1</th><th nowrap align=left>N2</th><th nowrap align=left>Np</th><th nowrap align=left>N optimal</th><th nowrap align=left>N conservative</th><th nowrap align=left>Optimal Set</th><th nowrap align=left>Conservative Set</th><th nowrap align=left>Rescue Ratio</th><th nowrap align=left>Self Consistency Ratio</th><th nowrap align=left>Reproducibility Test</th></tr>
<tr><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>0</td><td>self_pseudo_rep_rep1</td><td>self_pseudo_rep_rep1</td><td>0.0</td><td>1.0</td><td>pass</td></tr>
</table></div></div><br><div id='table_IDR-FRiP'><b>IDR FRiP</b><div style='padding-left: 20px'>
<table border='1'>
<tr><th nowrap align=left>Fraction of Reads in Peak</th></tr>
<tr><td>0</td></tr>
</table></div></div><br><div id='help-idr-FRiP'><p>ppr: Pooled pseudo replicate <br>rep1-pr : Pseudo replicates from replicate 1 <br>rep2-pr : Pseudo replicates from replicate 2</p><br><figure style='display: inline-block;'><img src='./peak/idr/pseudo_reps/rep1/COMBINED_TEST_rep1-pr.unthresholded-peaks.txt.png' width='800'><figcaption style='text-align: center;'><b>rep1 pseudo-reps</b></figcaption></figure>&nbsp</div><br><br></body></html>